subprojects/core/src/test/groovy/org/gradle/initialization/DefaultSettingsTest.groovy

```diff
@@ -95,6 +99,8 @@ class DefaultSettingsTest {
             will(returnValue(scriptHandlerFactory));
             allowing(settingsServices).get((Type)ProjectDescriptorRegistry.class);
             will(returnValue(projectDescriptorRegistry));
+            allowing(settingsServices).get(FeaturePreviews.class)
+            will(returnValue(previews))
```

Comment:
When you have to change one of these horrible old JMock tests, it's awesome if you take the time to convert the test to Spock. Definitely not required, but it feels good to clean up old stuff.

Reply:
I nearly asked about that, I should have. Will do, even if it ends up in a follow up PR.